### PR TITLE
gh-104360: remove reference to removed module-level wrap_socket

### PR DESCRIPTION
**Pull request by graingert**
* Issue: gh-104360

----
:books: Documentation preview :books:: https://cpython-previews--104361.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/ssl.rst [hunk 1]
```diff
@@ -2453,10 +2453,6 @@ provided.
      :exc:`SSLWantReadError` if it needs more data than the incoming BIO has
      available.
 
-   - There is no module-level ``wrap_bio()`` call like there is for
-     :meth:`~SSLContext.wrap_socket`. An :class:`SSLObject` is always created
-     via an :class:`SSLContext`.
-
    .. versionchanged:: 3.7
       :class:`SSLObject` instances must to created with
```
Comment (graingert) [inline Doc/library/ssl.rst]:
```suggestion
      :class:`SSLObject` instances must be created with
```

File: Doc/library/ssl.rst [hunk 2]
```diff
@@ -2453,12 +2453,8 @@ provided.
      :exc:`SSLWantReadError` if it needs more data than the incoming BIO has
      available.
 
-   - There is no module-level ``wrap_bio()`` call like there is for
```
Comment (eamanu) [inline Doc/library/ssl.rst:2477]: ssl.wrap_context() is removed, but ssl.SSLContext.wrap_socket does not.

Comment (graingert) [reply Doc/library/ssl.rst:2477]: @eamanu right but there is no module-level wrap_socket so this section doesn't make sense anymore

**Review by eamanu — APPROVED** [review]

**Review by AA-Turner — APPROVED** [review]
The substantive documentation was removed in #99023.

A

**Pull request merged**